src/frontend/components/DashboardVideoLiveControlPane/index.spec.tsx

```diff
@@ -1,10 +1,12 @@
+import { within } from '@testing-library/dom';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { ResponsiveContext } from 'grommet';
 import { DateTime } from 'luxon';
 import React from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 
```

Comment:
commit message: verb join is missing and spelling of disallow

✨(frontend) hide join discussion button for join mode closed
When the video join mode is closed, we want to **disallow** students to join
the discussion, so the button to ask to **join** is not displayed.